Components/Window.cs

```diff
@@ -315,7 +282,8 @@ public WindowSizeState GetWindowSizeState()
                 case NativeMethods.ShowWindowCommands.Minimize:
                 case NativeMethods.ShowWindowCommands.ShowMinimized:
                     return WindowSizeState.Minimized;
-                case NativeMethods.ShowWindowCommands.Maximize: // No need for ShowMaximized here since its also of value 3
+                case NativeMethods.ShowWindowCommands.Maximize
+                    : // No need for ShowMaximized here since its also of value 3
```

Comment:
```suggestion
                // No need for ShowMaximized here since its also of value 3
                case NativeMethods.ShowWindowCommands.Maximize:
```